### PR TITLE
[Doc Update][Extend a Teams personal tab across Microsoft 365]

### PR DESCRIPTION
**Pull request by JoshuaPartlow**
Added some clarifications to the _Extend a Teams personal tab across Microsoft 365_ article, as well as the associated pre-requisites topic based on feedback.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c8a94920c13c5555e7d239f219a2db25d0c125a3/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ad1579e-f28a-4723-9892-e2c6f54d570a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c8a94920c13c5555e7d239f219a2db25d0c125a3/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bfce7ad3-2d6a-42d8-8e5a-5581ae7a80d4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ To set up your development environment:
 
 You need a Microsoft 365 developer subscription sandbox tenant to set up your dev environment. If you don't already have one, create a [sandbox tenant](/office/developer-program/microsoft-365-developer-program-get-started) or get a test tenant through your organization.
 
+> [!NOTE]
+> As you set up for testing, make sure to sign-in with the account you intend to use for testing and sign-out from any others.
```
Comment (v-ypalikila) [inline msteams-platform/m365-apps/prerequisites.md:28]:
```suggestion
> [!NOTE]
> As you set up for testing, ensure that you sign out of all other accounts and sign in with the account you intend to use for testing.
```

File: msteams-platform/m365-apps/prerequisites.md [hunk 2]
```diff
@@ -71,7 +74,8 @@ You can preview Teams apps running in Outlook on Windows desktop by using a rece
 To install Office 365 Beta Channel applications in your test environment:
 
 1. Sign in to your test environment with your test tenant credentials.
-1. Download the [Office Deployment Tool](https://www.microsoft.com/download/details.aspx?id=49117) and extract to a local folder.
+1. Download and run the [Office Deployment Tool](https://www.microsoft.com/download/details.aspx?id=49117)
+1. Select a local folder to store the extracted files
```
Comment (v-ypalikila) [inline msteams-platform/m365-apps/prerequisites.md:78]:
```suggestion
1. Download and run the [Office Deployment Tool](https://www.microsoft.com/download/details.aspx?id=49117).
1. Select a local folder to store the extracted files.
```

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 3]
```diff
@@ -39,13 +39,13 @@ To complete this tutorial, you need:
 
 If you have an existing personal tab app, make a copy or a branch of your production project for testing and update your App ID in the app manifest to use a new identifier (distinct from the production App ID, for testing).
 
-If you'd like to use sample code to complete this tutorial, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend) to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, then return to this article to update it for Microsoft 365.
+If you'd like to use sample code to complete this tutorial, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend) to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, then return to this article to update it for Microsoft 365. You can also use the Teams Toolkit extension to create a new Teams app and select *Start from a sample*. One of the options available is the *Todo List with backend on Azure* sample.
```
Comment (v-ypalikila) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:42]:
```suggestion
If you'd like to use sample code to build a personal tab app using the Teams Toolkit extension for Visual Studio Code, follow the setup steps in the [Todo List Sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/main/todo-list-with-Azure-backend). After you've created a personal tab, return to this article to extend your personal tab across Microsoft 365. You can also use the Teams Toolkit extension to create a new Teams app and select *Start from a sample*. 
```

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 4]
```diff
@@ -39,13 +39,13 @@ To complete this tutorial, you need:
...
 
-Alternately, you can use a basic single sign-on *hello world* app already enabled Microsoft 365 in the following [Quickstart](#quickstart) section and then skip to [sideload your app in Teams](#sideload-your-app-in-teams).
+Alternately, you can use a basic single sign-on *hello world* app already enabled for Microsoft 365 in the following [Quickstart](#quickstart) section and then skip to [sideload your app in Teams](#sideload-your-app-in-teams).
 
 ### Quickstart
 
-To start with a [personal tab](https://github.com/OfficeDev/TeamsFx-Samples/tree/ga/todo-list-with-Azure-backend-M365) that's already enabled to run in Outlook and Office, use Teams Toolkit extension for Visual Studio Code.
+To start with a [personal tab](https://github.com/OfficeDev/TeamsFx-Samples/tree/ga/todo-list-with-Azure-backend-M365) that's already enabled to run in Outlook and Office, you can use the Teams Toolkit extension for Visual Studio Code.
```
Comment (v-ypalikila) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:48]: The link to personal tab here redirects to Azure-backend-M365. The Quickstart guide has the steps for single sign-on hello world app.

Request you to update the link to single sign-on hello world app.

**Comment by v-ypalikila** [comment]
Hi @JoshuaPartlow,

A gentle reminder. Please let us know if the changes are good and confirm if we can incorporate the changes and merge the PR.

Thanks,
Yash

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7022e1d9e68d8306ae32d9bb6a1cbab23669f849/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1048c299-fde4-4c1f-a277-c50c03a6f4e1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7022e1d9e68d8306ae32d9bb6a1cbab23669f849/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c4e4142-15a3-4b43-b9a0-2cb3e79ca130) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0055d1fe5f40ae5703d3db33494641c7e124a1c6/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/467c7198-109c-4c01-bc49-92e4017abf64) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0055d1fe5f40ae5703d3db33494641c7e124a1c6/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/52e1ab90-d0b4-4183-af02-e9d67f0a878b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 3 above)
Comment (JoshuaPartlow) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md]: This is more about using sample code vs your own production code. I've revised this section in the latest commit.

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 4 above)
Comment (JoshuaPartlow) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md]: Noted and removed link in the latest commit.

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1accc82eb674f732c40145d013e8c16087e008fa/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 95 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bfbdb583-18e0-4039-ab6d-d6d67e99625b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1accc82eb674f732c40145d013e8c16087e008fa/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 92 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eda6fe67-66b6-43b4-a690-45368a9de453) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**